src/services/userServices.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+let users = [];
+
+const init = () => {
+  users = [];
+};
+
+const getAll = () => {
+  return users;
+};
+
+const getUserById = (userId) => {
+  return users.find(user => +user.id === +userId) || null;
+};
+
+const create = (name) => {
+  const maxId = users.length > 0
+    ? Math.max(...users.map(({ id }) => id))
+    : 0;
+
+  const newUser = {
+    id: maxId + 1,
+    name,
+  };
```

Comment:
`maxId` it's a bad approach to creating a unique id, it's better to use the current Date, or use some library for making id, like 'uuid4'
check other places